### ISSUE TITLE
Unable to re-open docker container on remote mac via ssh

### ISSUE THREAD

**Issue by james-infinite-eye** [microsoft/vscode-remote-release#6532]
Everything was working perfectly fine until today, 

1. vscode connect via ssh to remote host, connects successful.
2. open project folder, then reopen in container
3. Error log:

```
[4884 ms] Remote-Containers 0.231.0 in VS Code 1.66.0 (e18005f0f1b33c29e81d732535d8c0e47cafb0b5).
[4883 ms] Start: Run: ssh 192.168.0.21 /bin/sh
[4895 ms] Start: Run in host: id -un
[5165 ms] 
james
-n 
[5165 ms] -n 
[5166 ms] Start: Run in host: cat /etc/passwd
[5173 ms] Start: Run in host: echo ~
[5177 ms] 
/Users/james
-n 
[5178 ms] -n 
[5178 ms] Start: Run in host: test -x '/Users/james/.vscode-remote-containers/bin/e18005f0f1b33c29e81d732535d8c0e47cafb0b5/node'
[5183 ms] 
-n 
[5183 ms] -n 
[5183 ms] Exit code 1
[5184 ms] Start: Run in host: test -x '/Users/james/.vscode-server/bin/e18005f0f1b33c29e81d732535d8c0e47cafb0b5/node'
[5187 ms] 
-n 
[5188 ms] -n 
[5188 ms] Start: Run in host: test -f '/Users/james/.vscode-server/bin/e18005f0f1b33c29e81d732535d8c0e47cafb0b5/node_modules/node-pty/package.json'
[5192 ms] 
-n 
[5192 ms] -n 
[5193 ms] Start: Run in host: test -f '/Users/james/.vscode-remote-containers/dist/vscode-remote-containers-server-0.231.0.js'
[5199 ms] 
-n 
[5199 ms] -n 
[5203 ms] Host server: -n 
[5203 ms] userEnvProbe: loginInteractiveShell (default)
[5204 ms] userEnvProbe shell: /bin/sh
[5285 ms] userEnvProbe PATHs:
Probe:     None
Container: None
[5290 ms] Start: Run in Host: docker version --format {{.Server.APIVersion}}
[5300 ms] Host server: node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error: spawn docker ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn docker',
  path: 'docker',
  spawnargs: [ 'version', '--format', '{{.Server.APIVersion}}' ]
}
[5304 ms] Host server: /bin/sh: line 10: name:[stdin]: command not found
[5305 ms] stream ended with:1 but wanted:9
[5308 ms] Host server: /bin/sh: line 11: name:[stdout]: No such file or directory
[5309 ms] Host server terminated (code: 1, signal: null).
```

Running this command when ssh'd into the mac i get the following:

```
docker version --format {{.Server.APIVersion}}
1.41
```

If i plug keyboard into the mac, i can open the folder in a remote container without issue.

**Comment by chrmarti** [comment]
I don't think Remote-Containers ever worked with a Mac as the SSH server. (Only Remote-SSH does.)

**Comment by adamduren** [comment]
@james-infinite-eye I rolled back to `ms-vscode-remote.remote-containers@v0.224.3` and it fixed the problem for me.

**Comment by james-infinite-eye** [comment]
@adamduren I htink i hit the same issue as you did, when changing to older extensions for Remote-Containsers, Remote-SSH 4weeks ago, i recieved the same error as #6529 , but it did start to open in a container.

**Comment by chrmarti** [comment]
@james-infinite-eye Could you append the log from a successful connection to a dev container with the old versions?

**Comment by james-infinite-eye** [comment]
```
[60 ms] Remote-Containers 0.224.3 in VS Code 1.65.2 (c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1).
[59 ms] Start: Resolving Remote
[66 ms] Start: Run: ssh 192.168.0.21 /bin/sh
[76 ms] Start: Run in host: id -un
[307 ms] 
james
-n 
[307 ms] -n 
[307 ms] Start: Run in host: cat /etc/passwd
[314 ms] Start: Run in host: echo ~
[318 ms] 
/Users/james
-n 
[318 ms] -n 
[319 ms] Start: Run in host: test -x '/Users/james/.vscode-remote-containers/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/node'
[322 ms] 
-n 
[322 ms] -n 
[322 ms] Exit code 1
[323 ms] Start: Run in host: test -x '/Users/james/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/node'
[327 ms] 
-n 
[327 ms] -n 
[327 ms] Start: Run in host: test -f '/Users/james/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/node_modules/node-pty/package.json'
[332 ms] 
-n 
[332 ms] -n 
[332 ms] Start: Run in host: test -f '/Users/james/.vscode-remote-containers/dist/vscode-remote-containers-server-0.224.3.js'
[337 ms] 
-n 
[337 ms] -n 
[340 ms] Host server: -n 
[341 ms] userEnvProbe: loginInteractiveShell (default)
[341 ms] userEnvProbe shell: /bin/sh
[440 ms] userEnvProbe PATHs:
Probe:     '/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin'
Container: None
[446 ms] Setting up container for folder or workspace: /Users/james/Projects/lanternhousearts.org
[446 ms] Host: ssh://192.168.0.21
[448 ms] Start: Check Docker is running
[448 ms] Start: Run in Host: docker version --format {{.Server.APIVersion}}
[574 ms] Server API version: 1.41
[615 ms] Start: Run in Host: docker-compose version --short
[981 ms] Start: Run in Host: docker ps -q -a --filter label=com.docker.compose.project=lanternhouseartsorg --filter label=com.docker.compose.service=php
[1122 ms] Start: Run in Host: docker inspect --type container c445ad1efe6d
[1260 ms] Start: Run in Host: docker-compose -f /Users/james/Projects/lanternhousearts.org/docker-compose.yml config
[1642 ms] services:
  adminer:
    build:
      context: /Users/james/Projects/lanternhousearts.org/.docker/adminer
    depends_on:
      db:
        condition: service_started
    links:
    - db
    ports:
    - published: 8081
      target: 8080
  db:
    command: --max_allowed_packet=500M
    environment:
      MYSQL_DATABASE: wordpress
      MYSQL_PASSWORD: pass
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: wp_user
    expose:
    - '3306'
    image: mysql:5.7
    ports:
    - published: 3306
      target: 3306
    volumes:
    - /Users/james/Projects/lanternhousearts.org/.docker/data/mysql:/var/lib/mysql:rw
  mailcatcher:
    build:
      context: /Users/james/Projects/lanternhousearts.org/.docker/mailcatcher
    ports:
    - published: 1025
      target: 1025
    - published: 1080
      target: 1080
    volumes:
    - /Users/james/Projects/lanternhousearts.org/.docker/data/mailcatcher:/data:rw
  php:
    build:
      context: /Users/james/Projects/lanternhousearts.org/.docker/wordpress
    depends_on:
      db:
        condition: service_started
    environment:
      WORDPRESS_DB_HOST: db:3306
      WORDPRESS_DB_PASSWORD: pass
      WORDPRESS_DB_USER: wp_user
    expose:
    - '9000'
    links:
    - db
    ports:
    - published: 80
      target: 80
    - published: 443
      target: 443
    - published: 9000
      target: 9000
    volumes:
    - /Users/james/Projects/lanternhousearts.org/spektrix-plugin:/var/www/html/wp-content/plugins/lanternhouse-spektrix:rw
    - /Users/james/Projects/lanternhousearts.org/gutenberg-plugin:/var/www/html/wp-content/plugins/lanternhouse:rw
    - /Users/james/Projects/lanternhousearts.org/src:/var/www/html/wp-content/themes/infinite-eye:rw
    - node_modules:/var/www/html/wp-content/themes/infinite-eye/node_modules:rw
    - /Users/james/Projects/lanternhousearts.org/deploy-config.php:/var/www/deploy-config.php:rw
    - /Users/james/Projects/lanternhousearts.org/deploy.php:/var/www/html/deploy.php:rw
version: '3'
volumes:
  node_modules: {}

[1646 ms] Start: Run in Host: docker -v
[1718 ms] Start: Run in Host: docker events --format {{json .}} --filter event=start
[1725 ms] PersistedPath=/Users/james/.vscode-remote-containers/data, ContainerHasLabels=true
[1725 ms] Container was created with these config files: /Users/james/Projects/lanternhousearts.org/docker-compose.yml,/Users/james/.vscode-remote-containers/data/docker-compose/docker-compose.devcontainer.containerFeatures-1647966024380.yml
[1734 ms] Expected /Users/james/.vscode-remote-containers/data/docker-compose/docker-compose.devcontainer.containerFeatures-1647966024380.yml to exist, but it did not
[1735 ms] Generating composeOverrideFile...
[1745 ms] Start: Run in Host: docker inspect --type image lanternhouseartsorg_php
[1903 ms] Writing docker-compose.devcontainer.containerFeatures-1648736661879.yml to /Users/james/.vscode-remote-containers/data/docker-compose
[1921 ms] Start: Run in Host: docker-compose --project-name lanternhouseartsorg -f /Users/james/Projects/lanternhousearts.org/docker-compose.yml -f /Users/james/.vscode-remote-containers/data/docker-compose/docker-compose.devcontainer.containerFeatures-1648736661879.yml up -d
Starting lanternhouseartsorg_db_1          ... done
Starting lanternhouseartsorg_mailcatcher_1 ... done
Starting lanternhouseartsorg_php_1         ... done
Starting lanternhouseartsorg_adminer_1     ... done
[3680 ms] Start: Run in Host: docker ps -q -a --filter label=com.docker.compose.project=lanternhouseartsorg --filter label=com.docker.compose.service=php
[3829 ms] Start: Run in Host: docker inspect --type container c445ad1efe6d
[3979 ms] Start: Inspecting container
[3979 ms] Start: Run in Host: docker inspect --type container c445ad1efe6df72dc0eff80a794ea0a984d7fc0e126b717fcd4f1fd116cae340
[4132 ms] Start: Run in Host: docker exec -i -u root -e VSCODE_REMOTE_CONTAINERS_SESSION=6ee523b8-598f-492c-be52-79f1b33c7d9e1648736659142 c445ad1efe6df72dc0eff80a794ea0a984d7fc0e126b717fcd4f1fd116cae340 /bin/sh
[4137 ms] Start: Run in container: uname -m
[4376 ms] x86_64
[4376 ms] 
[4376 ms] Start: Run in container: (cat /etc/os-release || cat /usr/lib/os-release) 2>/dev/null
[4383 ms] PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
[4383 ms] 
[4383 ms] Start: Run in container: cat /etc/passwd
[4390 ms] Start: Setup shutdown monitor
[4391 ms] Forking shutdown monitor: c:\Users\james\.vscode\extensions\ms-vscode-remote.remote-containers-0.224.3\dist\shutdown\shutdownMonitorProcess \\.\pipe\vscode-remote-containers-ac28f21b86a57e8e2ac4d83930446e6d6044a5e3-sock dockerCompose Debug c:\Users\james\AppData\Roaming\Code\logs\20220331T152109\exthost1\ms-vscode-remote.remote-containers 1648736659976
[4400 ms] Start: Run in container: test -d /root/.vscode-server
[4409 ms] 
[4409 ms] 
[4410 ms] Start: Run in container: test -f '/var/vscode-server/.patchEtcEnvironmentMarker'
[4417 ms] 
[4417 ms] 
[4418 ms] Start: Run in container: test -f '/var/vscode-server/.patchEtcProfileMarker'
[4425 ms] 
[4426 ms] 
[4426 ms] Start: Run in container: test ! -f '/root/.vscode-server/data/Machine/.writeMachineSettingsMarker' && set -o noclobber && mkdir -p '/root/.vscode-server/data/Machine' && { > '/root/.vscode-server/data/Machine/.writeMachineSettingsMarker' ; } 2> /dev/null
[4433 ms] 
[4433 ms] 
[4433 ms] Exit code 1
[4433 ms] Start: Run in container: cat /root/.vscode-server/data/Machine/settings.json
[4441 ms] {
        "editor.formatOnSave": true,
        "eslint.packageManager": "yarn",
        "eslint.alwaysShowStatus": true,
        "npm.includeDirectories": [
                "wp-content/themes/infinite-eye"
        ],
        "npm.useRootDirectory": false,
        "npm.bin": "yarn",
        "npm.runInTerminal": true,
        "prettier.singleQuote": true,
        "editor.fontLigatures": true,
        "editor.fontFamily": "Fira Code Light",
        "editor.fontSize": 12
}
[4441 ms] 
[4441 ms] Start: Run in container: test -d /root/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1
[4447 ms] 
[4447 ms] 
[4448 ms] Start: Launching Remote-Containers helper.
[4448 ms] ssh-agent: SSH_AUTH_SOCK in container (/tmp/vscode-ssh-auth-a89832ce9d4539945c0aeec49f0bbddfcfe60695.sock) forwarded to local host (\\.\pipe\openssh-ssh-agent).
[4448 ms] Start: Run in Host: gpgconf --list-dir agent-extra-socket
[4459 ms] Host server: (node:7335) UnhandledPromiseRejectionWarning: Error: spawn gpgconf ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
(Use `node --trace-warnings ...` to show where the warning was created)
[4460 ms] Start: Run: gpgconf --list-dir agent-extra-socket
[4460 ms] Host server: (node:7335) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7335) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[4466 ms] Host server: (node:7335) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
[4475 ms] findLocalWindowsExecutable: Exectuable 'gpgconf' not found on PATH 'C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\nodejs\;C:\Program Files\dotnet\;C:\Program Files\Git\cmd;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Docker\Docker\resources\bin;C:\ProgramData\DockerDesktop\version-bin;C:\Program Files\Microsoft VS Code\bin;C:\Users\james\AppData\Local\Microsoft\WindowsApps;C:\Users\james\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\james\AppData\Roaming\npm;C:\Users\james\AppData\Local\Android\Sdk\tools\bin;C:\Users\james\AppData\Local\Android\Sdk\platform-tools;C:\Users\james\AppData\Local\Android\Sdk\emulator;'.
[4477 ms] Start: Run in Host: docker exec -i -u root c445ad1efe6df72dc0eff80a794ea0a984d7fc0e126b717fcd4f1fd116cae340 /bin/sh
[4477 ms] userEnvProbe: loginInteractiveShell (default)
[4478 ms] userEnvProbe shell: /bin/bash
[4484 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-a89832ce9d4539945c0aeec49f0bbddfcfe60695.js' >/tmp/vscode-remote-containers-a89832ce9d4539945c0aeec49f0bbddfcfe60695.js
[4491 ms] Start: Run in container: # Test for /root/.ssh/known_hosts and ssh
[4502 ms] /root/.ssh/known_hosts exists
[4502 ms] 
[4502 ms] Exit code 1
[4502 ms] Start: Run in container: command -v git >/dev/null 2>&1 && git config --system --replace-all credential.helper '!f() { /root/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/node /tmp/vscode-remote-containers-a89832ce9d4539945c0aeec49f0bbddfcfe60695.js $*; }; f' || true
[4509 ms] 
[4509 ms] 
[4510 ms] Start: Preparing Extensions
[4510 ms] Start: Run in container: test ! -f '/root/.vscode-server/data/Machine/.installExtensionsMarker' && set -o noclobber && mkdir -p '/root/.vscode-server/data/Machine' && { > '/root/.vscode-server/data/Machine/.installExtensionsMarker' ; } 2> /dev/null
[4515 ms] 
[4515 ms] 
[4516 ms] Exit code 1
[4517 ms] Extensions cache, install extensions: None
[4517 ms] Start: Run in container: test -d /root/.vscode-server/extensionsCache && ls /root/.vscode-server/extensionsCache || true
[4533 ms] bmewburn.vscode-intelephense-client-1.8.2
esbenp.prettier-vscode-9.3.0
esbenp.prettier-vscode-9.5.0
janisdd.vscode-edit-csv-0.6.6
janisdd.vscode-edit-csv-0.6.8
[4533 ms] 
[4534 ms] Extensions cache, copy to remote: None
[4535 ms] Start: Run in container: for pid in `cd /proc && ls -d [0-9]*`; do { echo $pid ; readlink /proc/$pid/cwd ; readlink /proc/$pid/ns/mnt ; cat /proc/$pid/stat | tr "
[4597 ms] Start: Run in container: cat '/root/.vscode-server/data/Machine/.connection-token-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1' 2>/dev/null || (umask 377 && echo '6289b8e6-0d55-47ce-9b6e-7c11d5c27558' >'/root/.vscode-server/data/Machine/.connection-token-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1-dd96da7f-69b0-4ad5-aa5e-78bfd93ce67a' && mv -n '/root/.vscode-server/data/Machine/.connection-token-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1-dd96da7f-69b0-4ad5-aa5e-78bfd93ce67a' '/root/.vscode-server/data/Machine/.connection-token-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1' && rm -f '/root/.vscode-server/data/Machine/.connection-token-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1-dd96da7f-69b0-4ad5-aa5e-78bfd93ce67a' && cat '/root/.vscode-server/data/Machine/.connection-token-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1')
[4606 ms] 7536cb40-93d3-444b-94ca-91d776cab441
[4606 ms] 
[4606 ms] Start: Starting VS Code Server
[4696 ms] 
[4696 ms] 
[4696 ms] Start: Run in container: cat <<'EOF-/tmp/vscode-remote-containers-server-a89832ce9d4539945c0aeec49f0bbddfcfe60695.js' >/tmp/vscode-remote-containers-server-a89832ce9d4539945c0aeec49f0bbddfcfe60695.js_1648736664672
[4723 ms] 
[4723 ms] 
[4729 ms] Start: Run in Container: /root/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/bin/code-server --log debug --force-disable-user-env --server-data-dir /root/.vscode-server --accept-server-license-terms --host 127.0.0.1 --port 0 --connection-token-file /root/.vscode-server/data/Machine/.connection-token-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1 --extensions-download-dir /root/.vscode-server/extensionsCache --start-server 
[4872 ms] userEnvProbe PATHs:
Probe:     '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
Container: '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
[4876 ms] *
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
[4882 ms] Server bound to 127.0.0.1:38649 (IPv4)
Extension host agent listening on 38649

[4882 ms] Start: Run in container: echo 38649 >'/root/.vscode-server/data/Machine/.devport-c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1'
[4887 ms] 
[4888 ms] 
[4888 ms] Port forwarding for container port 38649 starts listening on local port.
[4889 ms] Port forwarding local port 38649 to container port 38649
[4890 ms] Start: Run in container: mkdir -p '/root/.vscode-server/data/Machine' && CONTENT="$(cat '/root/.vscode-server/data/Machine/.onCreateCommandMarker' 2>/dev/null || echo ENOENT)" && [ "${CONTENT:-2022-03-22T16:20:25.226731857Z}" != '2022-03-22T16:20:25.226731857Z' ] && echo '2022-03-22T16:20:25.226731857Z' > '/root/.vscode-server/data/Machine/.onCreateCommandMarker'
[4898 ms] 
[4898 ms] 
[4898 ms] Exit code 1
[4899 ms] Start: Run in container: mkdir -p '/root/.vscode-server/data/Machine' && CONTENT="$(cat '/root/.vscode-server/data/Machine/.updateContentCommandMarker' 2>/dev/null || echo ENOENT)" && [ "${CONTENT:-2022-03-22T16:20:25.226731857Z}" != '2022-03-22T16:20:25.226731857Z' ] && echo '2022-03-22T16:20:25.226731857Z' > '/root/.vscode-server/data/Machine/.updateContentCommandMarker'
[4907 ms] 
[4907 ms] 
[4907 ms] Exit code 1
[4908 ms] Start: Run in container: mkdir -p '/root/.vscode-server/data/Machine' && CONTENT="$(cat '/root/.vscode-server/data/Machine/.postCreateCommandMarker' 2>/dev/null || echo ENOENT)" && [ "${CONTENT:-2022-03-22T16:20:25.226731857Z}" != '2022-03-22T16:20:25.226731857Z' ] && echo '2022-03-22T16:20:25.226731857Z' > '/root/.vscode-server/data/Machine/.postCreateCommandMarker'
[4908 ms] 

------------------------
Published Ports:
443/tcp -> 0.0.0.0:443
80/tcp -> 0.0.0.0:80
9000/tcp -> 0.0.0.0:9000
------------------------

[4915 ms] Port forwarding connection from 64483 > 38649 > 38649 in the container.
[4915 ms] Start: Run in Host: docker exec -i -u root -e VSCODE_REMOTE_CONTAINERS_SESSION=6ee523b8-598f-492c-be52-79f1b33c7d9e1648736659142 c445ad1efe6df72dc0eff80a794ea0a984d7fc0e126b717fcd4f1fd116cae340 /root/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/node -e 
[4917 ms] 
[4917 ms] 
[4918 ms] Exit code 1
[4933 ms] Start: Run in container: # Test for /root/.gitconfig and git
[4941 ms] /root/.gitconfig exists
[4941 ms] 
[4941 ms] Exit code 1
[4950 ms] Start: Run in container: command -v git >/dev/null 2>&1 && git config --global --replace-all credential.helper '!f() { /root/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/node /tmp/vscode-remote-containers-a89832ce9d4539945c0aeec49f0bbddfcfe60695.js $*; }; f' || true
[4957 ms] 
[4958 ms] 
[4958 ms] Start: Run in container: mkdir -p '/root/.vscode-server/data/Machine' && CONTENT="$(cat '/root/.vscode-server/data/Machine/.postStartCommandMarker' 2>/dev/null || echo ENOENT)" && [ "${CONTENT:-2022-03-31T14:24:00.323869Z}" != '2022-03-31T14:24:00.323869Z' ] && echo '2022-03-31T14:24:00.323869Z' > '/root/.vscode-server/data/Machine/.postStartCommandMarker'
[4968 ms] 
[4968 ms] 
[5221 ms] [14:24:01] Extension host agent started.
[5276 ms] Port forwarding connection from 54584 > 38649 > 38649 in the container.
[5277 ms] Start: Run in Host: docker exec -i -u root -e VSCODE_REMOTE_CONTAINERS_SESSION=6ee523b8-598f-492c-be52-79f1b33c7d9e1648736659142 c445ad1efe6df72dc0eff80a794ea0a984d7fc0e126b717fcd4f1fd116cae340 /root/.vscode-server/bin/c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1/node -e 
[5332 ms] [14:24:02] [127.0.0.1][f292398f][ManagementConnection] New connection established.
[5338 ms] [14:24:02] Deleted from disk esbenp.prettier-vscode /root/.vscode-server/extensions/esbenp.prettier-vscode-9.3.0
[5341 ms] [14:24:02] Deleted from disk janisdd.vscode-edit-csv /root/.vscode-server/extensions/janisdd.vscode-edit-csv-0.6.6
[5672 ms] [14:24:02] [127.0.0.1][6a59f8a7][ExtensionHostConnection] New connection established.
[5678 ms] [14:24:02] [127.0.0.1][6a59f8a7][ExtensionHostConnection] <259> Launched Extension Host Process.
```

**Comment by chrmarti** [comment]
Note that v0.231.4 is missing a change to keep connecting through a Mac working. It is fixed in v0.232.4-pre-release and the fix will be in the next stable release. Sorry for the inconvenience.
